File: Project Euler #6/Python/solution6.py
if __name__ == '__main__':
    t = int(input().strip())
    for a0 in range(t):
        n = int(input().strip())
        sum1 = n*(n+1)//2
        sum2 = n*(n+1)*(2*n+1)//6
        print(sum1*sum1 - sum2);

File: Project Euler #6/CPP/solution6.cpp
#include <iostream>

using namespace std;

int main() {
      
    int t;
    cin>>t;
    while(t--){
        int n;
        cin>>n;
        long long sum1 = (long)n*(n+1)/2;
        long long sum2 = (long)n*(n+1)*(2*n+1)/6;
        cout<<sum1*sum1-sum2<<endl;
    }
    return 0;
}
